communication/marketing-team/social-guidelines.md

```diff
@@ -24,19 +24,42 @@ Kubernetes Organization Members with access to account are:
 ## Themes/Topics
 
 - Contributor/SIG blog posts
-- Events
-- KEPs
+- Contributor events
+- KEPs*
+  - see [#KEPs] section for further guidance
 - Process changes
 - Release info, code freeze, et al
 - Accomplishments (reaching 100,000 GitHub stars for example)
 - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
+- Steering Committee Announcements
+
+## KEPs
+([Kubernetes Enhancement Proposals])
+
+### Ground Rules  
+- Social Media team can not proactively look for KEPs to amplify.
+- It's not ok to tweet about a KEP that is going to status: implementable until it's been merged.
```

Comment:
Are all KEP status' eligible for Tweeting? Or is it only `status: implementable`? Similarly, are all enhancement states (alpha,beta,stable) also eligible?

Reply:
I like calling out whats not eligible from a status perspective